cli/src/lib/hashing/tokenHash.ts

```diff
@@ -0,0 +1,17 @@
+export class TokenHash {
+  private readonly mod: number;
+  private readonly base: number;
+
+  constructor() {
+    this.mod = 33554467;
+    this.base = 5801;
+  }
+
+  public hashToken(token: string): number {
+    let hash = 0;
+    for (let i = 0; i < token.length; i++) {
+      hash = (this.base * hash + token.charCodeAt(i)) % this.mod;
+    }
+    return hash;
```

Comment:
@toonijn can you check if this makes sense as a hashing algorithm? This is essentially the same hashing algorithm as the one used in [rollingHash.ts](https://github.com/dodona-edu/dolos/blob/hash-tokens/cli/src/lib/hashing/rollingHash.ts#L35-L42), except it is not rolling.

This will be used to hash tokens, which will usuallyl be very short (< 20 chars) strings.